### ISSUE TITLE
Change pcc reporting to separate logits vs past_key_values

### ISSUE THREAD

**Issue by ddilbazTT** [tenstorrent/tt-xla#3959]
Based off of https://github.com/tenstorrent/tt-xla/issues/3586

GPT OSS logits pcc is much better than what's reported overall due to past key values. Need to change either of these:
- add `decode_outputs` function to gpt_oss
- change reported pcc for models so the pcc check happens on logits instead of past key values
